Sources/TuistGenerator/Utils/SettingsHelper.swift

```diff
@@ -44,4 +38,34 @@ final class SettingsHelper {
     func variant(_ buildConfiguration: BuildConfiguration) -> BuildSettingsProvider.Variant {
         return buildConfiguration.variant == .debug ? .debug : .release
     }
+
+    // MARK: - Private
+
+    private func resolveValue(oldValue: SettingValue?, newValue: SettingValue) -> SettingValue {
```

Comment:
I think `resolveValue` is a misleading name here. What about using something like merge?

Reply:
+1 for `merge`, updated👍